CopilotKit/packages/runtime/src/service-adapters/index.ts

```diff
@@ -1,4 +1,8 @@
-export type { CopilotServiceAdapter } from "./service-adapter";
+export type {
+  CopilotRuntimeChatCompletionRequest,
```

Comment:
These two extra so someone can define their own adapter easier (I had to mock these classes). One missing piece is the `randomId` in `shared`, that's still inaccessible from outside (if I understand correctly)